_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -64,9 +66,9 @@ Both of the above two sections have been focused on increasing flow of business
+If the team requires a skill that it is missing, this is where an enabling team comes it. It exists temporarily and collaborates with the team for a short time, in order to get the product team up to speed. During this time, the interaction method would be one of collaboration (this is in contrast with the interaction mode between teams in Figure 1, which is XaaS). If the team requires some functionality that requires very difficult skill sets than the ones existing within the team, this may be a natural fracture plane whereby one could create a separate team, and then have the teams communicate via XaaS rather than through direct collaboration in an enabling capacity. For further discussion regarding the details of this approach, it is recommended to read Team Topologies, referenced to and linked to above.
 
-It is for this reason, enabling flow within the team and upskilling the team to have the skills required to deliver business value, that champions were designed (listed below). The idea of a champion is to have a member of the team focus on a given area and be dedicated to its improvement. For example, the accessibility champion would be responsible for (not accountable for, as the whole team is always responsible for the whole product) identifying and ideally rectifying issues pertaining to their focus. Ideally, if there is a team dedicated to accessibility, for example, the champion should reach out in order to identify existing best practices within the organization. Another option would have been to onboard an accessibility resource (using accessibility as an example, the same is true for any competency the team is lacking), however, we had difficulties finding a dedicated accessibility resource given the circumstances, so a champion was leveraged instead.
+It is for this reason, enabling flow within the team and upskilling the team to have the skills required to deliver business value, that champions were assigned (listed below). The idea of a champion is to have a member of the team focus on a given area and be dedicated to its improvement. For example, the accessibility champion would be responsible for (not accountable for, as the whole team is always responsible for the whole product) identifying and ideally rectifying issues pertaining to their focus. Ideally, if there is a team dedicated to accessibility, for example, the champion should reach out in order to identify existing best practices within the organization. Another option would have been to onboard an accessibility resource (using accessibility as an example, the same is true for any competency the team is lacking), however, we had difficulties finding a dedicated accessibility resource given the circumstances, so a champion was leveraged instead.
```

Comment:
Could sentence one be simplified as follows "Champions can assist with this* and for the HR whitelisting app they were assigned as listed below."
* I think saying 'this' is ok and means avoiding duplication of text from the above paragraph

Feels like the last few sentences are a sensitive area...Also is a bit confusing as we are saying champions are good but then the last sentence seems to say it would have been better to have an accessibility resource but as a backup we used the champion instead?

Reply:
This is true. I changed the phrasing 

" ... so we designated a team member who was interested in a11y to dedicate time to learning and improving the a11y of the application."